frontend/src/App.svelte

```diff
@@ -22,8 +18,12 @@
             <Home />
         </Route>
 
-        <Route path="/profile">
-            <Profile />
+        <Route path="/profile/me">
+            <LoggedInUserProfile />
+        </Route>
+
+        <Route path="/profile/:userId" let:params>
+            <UserProfileById userId={parseInt(params.userId)} />
```

Comment:
it's always prudent to add a radix when using `parseInt`...

Reply:
True